backend/sendEmail.py

```diff
@@ -0,0 +1,38 @@
+import base64
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from pickle import FALSE, TRUE
+import smtplib
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+class sendEmail:
+    
+    
+    def __init__(self, email):
+        self.ourEmail = os.getenv('OUR_EMAIL')
+        self.password = os.getenv('EMAIL_PASS')
+        self.recipient_email = email
+        self.subject = "Reset Password."
+    
+    def send(self):
+        message = f'Subject: {self.subject } \n\n Click the link to change your password.\n http:127.0.0.1:5000/password/reset'
```

Comment:
Does this mean we are sending an email for them to access the forgot password page ?